pixi.toml

```diff
@@ -0,0 +1,19 @@
+[project]
+name = "icub-tests"
+version = "0.0.0"
+description = "Set of tests to check the functionalities of a robot"
+authors = ["Martina Gloria <martina.gloria@iit.it>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "win-64", "osx-arm64", "osx-64"]
+
+[tasks]
+build_imu_test = {cmd = "cmake -S. -B.build -DICUB_TESTS_COMPILES_IMU_TEST=ON"}
+compile_imu_test = {cmd = "cmake --build .build", depends_on = ["build_imu_test"]}
```

Comment:
Are you sure that this works? If you are not installing the repo, how does the `robottestingframework-testrunner` find the tests?

Reply:
You're right, obviously this doesn't work if robottestingframework is not installed. I think that it'll be the same for the other dependencies, such as yarp, robometry and idyntree. In this case, to install the dependencies it would be sufficient to clone and compile them as one or multiple pixi tasks?

Reply:
You're right, obviously this doesn't work if robottestingframework is not installed. In this case, to install the dependencies it would be sufficient to clone and compile them as one or multiple pixi tasks?

Reply:
Actually `robottestingframework-testrunner` will work fine (it is installed by the `robot-testing-framework` package, see https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Frobot-testing-framework-2.0.1-hcb278e6_1.conda), what is missing is that you need to install the icub-tests, or let somehow know to RTF how to load the plugins compiled by this project. If I recall correctly, you need to do this by setting some env variable, see https://github.com/robotology/robotology-superbuild/blob/0215a26481e12fb2c46acf54492c2852c88a8e35/doc/environment-variables-configuration.md?plain=1#L29 .